application/templates/hpa.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Values.autoscaling.enabled .Values.deployment }}
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ template "application.name" . }}
+  labels:
+    {{ include "application.labels.stakater" . | nindent 4 }}
+    {{ include "application.labels.chart" . | nindent 4 }}
+  {{- if .Values.autoscaling.additionalLabels }}
+    {{ toYaml .Values.autoscaling.additionalLabels | nindent 4 }}
```

Comment:
If we add `-` like this  {{- toYaml .Values.autoscaling.additionalLabels | nindent 4 }} additional space will not be added during templating